nemo/collections/nlp/modules/common/huggingface/huggingface_utils.py

```diff
@@ -64,6 +64,13 @@
     },
 }
 
+VOCAB_FILE_NAME = {
```

Comment:
Can this be avoided? is it possible to return the path with `save_vocabulary`?

Reply:
The different models mean different things by "vocab_file", compare: https://github.com/huggingface/transformers/blob/master/src/transformers/models/albert/tokenization_albert.py#L33 vs https://github.com/huggingface/transformers/blob/master/src/transformers/models/bert/tokenization_bert.py#L32. I think it's better to preserve this info for the users

Reply:
It's fine to preserve it, but can we add the functionality to save_vocabulary. Also, what happens if the tokenizer_name is not in this dictionary?